.github/workflows/build-pr.yml

```diff
@@ -1,8 +1,6 @@
 name: Build
 
 on:
-  push:
```

Comment:
why is this removed now?
Is it because there is a new file now: "build-push" that superseeds it?

Reply:
To have a different pipeline run on PR vs push to master (so we don't run the Docker tests for every PR build), I had to separate them into two files, `build-pr` and `build-push`.

Reply:
cool. that's what I thought. thanks